backend/src/main/java/ai/verta/modeldb/versioning/blob/diff/DiffMerger.java

```diff
@@ -70,7 +70,10 @@ else if (!elA.contains(diffB)) {
                         // TODO: error otherwise
                         if (diffA != null && diffB != null) {
                           if (!elA.contains(diffA)) { // diff is applied on a state different from A
-                            conflictKeys.add(key);
+                            if (!elA.contains(
```

Comment:
when the end modification was exactly the same we would still call a conflict , this fixes that case